src/main/java/server/request/RequestParser.java

```diff
@@ -0,0 +1,37 @@
+package main.java.server.request;
+
+import java.util.HashMap;
+
+public class RequestParser {
+
+    private String[] segmentedRequest;
+    private final HashMap<String, String> headers = new HashMap<>();
+
+    public Request create(String incomingRequest) {
+        this.segmentedRequest  = incomingRequest.split("\r\n")[0].split(" ");
```

Comment:
This definitely makes me antsy about what happens if you get a malformed request. Maybe we should create a story for implementing a `400` response?